install/infra/modules/tools/issuer/variables.tf

```diff
@@ -11,6 +11,10 @@ variable "secretAccessKey" {
     default     = null
 }
 
+variable "gcp_credentials" {
```

Comment:
**question:** I see that we're defining a `gcp_credentials` variable here, how does this relate to the `data.local_file.gcp-credentials` datasource used in the kubernetes secret above?

Reply:
It is a data block that reads the content of the file provided. Actually, your comment made me realized that this secret is being created absolutely in the wrong place. This is necessary only for GCP. I will move this to the gcp `dns.tf` file.

Reply:
This has been changed too. Now that GKE module is returning keys instead of file names, we pass directly key to this module.